frontend/src/components/projects/myProjectNav.js

```diff
@@ -143,42 +143,52 @@ export const MyProjectNav = (props) => {
               </>
             )}
             {props.management && (userDetails.role === 'ADMIN' || isOrgManager) && (
-              <>
-                <FilterButton
-                  query={fullProjectsQuery}
-                  newQueryParams={{ status: 'PUBLISHED', managedByMe: 1, createdByMe: undefined }}
-                  setQuery={setQuery}
-                  isActive={
-                    fullProjectsQuery.managedByMe && fullProjectsQuery.status === 'PUBLISHED'
-                  }
-                >
-                  <FormattedMessage {...messages.active} />
-                </FilterButton>
-                <FilterButton
-                  query={fullProjectsQuery}
-                  newQueryParams={{ status: 'DRAFT', managedByMe: 1, createdByMe: undefined }}
-                  setQuery={setQuery}
-                  isActive={isActiveButton('DRAFT', fullProjectsQuery)}
-                >
-                  <FormattedMessage {...messages.draft} />
-                </FilterButton>
-                <FilterButton
-                  query={fullProjectsQuery}
-                  newQueryParams={{ status: 'ARCHIVED', managedByMe: 1, createdByMe: undefined }}
-                  setQuery={setQuery}
-                  isActive={isActiveButton('ARCHIVED', fullProjectsQuery)}
-                >
-                  <FormattedMessage {...messages.archived} />
-                </FilterButton>
-                <FilterButton
-                  query={fullProjectsQuery}
-                  newQueryParams={{ status: undefined, managedByMe: undefined, createdByMe: 1 }}
-                  setQuery={setQuery}
-                  isActive={isActiveButton('createdByMe', fullProjectsQuery)}
-                >
-                  <FormattedMessage {...messages.created} />
-                </FilterButton>
-              </>
+              <div>
+                <div className="dib pr4">
+                  <FilterButton
+                    query={fullProjectsQuery}
+                    newQueryParams={{ status: 'PUBLISHED' }}
+                    setQuery={setQuery}
+                    isActive={isActiveButton('PUBLISHED', fullProjectsQuery)}
```

Comment:
I removed the check for `fullProjectsQuery.managedByMe` here in the previous version of the code because if not, you won't be able to select this filter for projects "Created by Me". And it'll look wonky and confusing on the UI too. 

So I thought that the best way here was letting the user have the full freedom over how they want to filter stuff by letting this filter follow the behaviour of the other filters at the same level.